frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard-footer.tsx

```diff
@@ -118,7 +121,7 @@ const CreateVMWizardFooterComponent: React.FC<CreateVMWizardFooterComponentProps
         const isLastStep = activeStepID === VMWizardTab.RESULT;
 
         const isNextButtonDisabled = isAnyStepLocked;
-        const isReviewButtonDisabled = isAnyStepLocked;
+        const isReviewButtonDisabled = isAnyStepLocked || isInvalidUserTemplate;
```

Comment:
this should also apply for the create button. Can you create isCreateButtonDisabed variable?

and apply the same logic like we do for the render?

quote:
```tsx
             {activeStepID === VMWizardTab.REVIEW
                  ? getCreateVMLikeEntityLabel(isCreateTemplate, isProviderImport)
                  : 'Next'}
```